.github/workflows/tests.yml

```diff
@@ -16,21 +16,18 @@ jobs:
   test_package:
     name: Tests
     runs-on: ubuntu-latest
+
     steps:
-      - uses: actions/checkout@v4
-      - name: Setup python
-        uses: actions/setup-python@v5
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
         with:
-          python-version: "3.11"
-          architecture: x64
-          cache: pip
-          cache-dependency-path: "**/pyproject.toml"
-      - name: Install Poetry
-        run: |
-          pip install poetry
-      - name: Install dependencies
-        run: |
-          poetry install
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Are there any concerns about this running for forks/external contibutions? Or is this completely hidden from them?

Reply:
I believe that forks don't have access to secrets at all, so this would fail